sherpa/cpp_api/grpc/CMakeLists.txt

```
@@ -0,0 +1,115 @@
+add_definitions(-DASIO_STANDALONE)
+
+# compile sherpo.proto
+set(PROTO_DIR "${CMAKE_CURRENT_SOURCE_DIR}")
```

```suggestion
set(PROTO_DIR "${CMAKE_CURRENT_BINARY_DIR}")
include_directories(${PROTO_DIR})
```

We usually put generated files in the build directory.